medusa/server/api/v1/core.py

```diff
@@ -192,11 +197,11 @@ def call_dispatcher(self, args, kwargs):  # pylint:disable=too-many-branches
                         elif _is_int(cmd):
                             cur_out_dict = TVDBShorthandWrapper(cur_args, cur_kwargs, cmd).run()
                         else:
-                            cur_out_dict = _responds(RESULT_ERROR, "No such cmd: '" + cmd + "'")
+                            cur_out_dict = _responds(RESULT_ERROR, 'No such cmd: {0!r}'.format(cmd))
                     except ApiError as error:  # Api errors that we raised, they are harmless
-                        cur_out_dict = _responds(RESULT_ERROR, msg=ex(error))
+                        cur_out_dict = _responds(RESULT_ERROR, msg=error)
                 else:  # if someone chained one of the forbidden commands they will get an error for this one cmd
-                    cur_out_dict = _responds(RESULT_ERROR, msg="The cmd '" + cmd + "' is not supported while chaining")
+                    cur_out_dict = _responds(RESULT_ERROR, msg='The cmd {0!r} is not supported while chaining'.fomrat(cmd))
```

Comment:
I think this is the only critical change needed. After that we can start testing it. @p0psicles?